test/controllers/util/FormFieldSpec.scala

```diff
@@ -75,52 +75,6 @@ class FormFieldSpec extends AnyWordSpec with MockitoSugar with BeforeAndAfterEac
         .fieldErrors shouldBe List("alternativeDesc must be 8000 characters or less")
       RadioButtonGroupField.updateError(radioButtonGroupField, CLOSURE_STATUS_IS_MISSING).fieldErrors shouldBe List("CLOSURE_STATUS_IS_MISSING")
     }
-
```

Comment:
Unused dependency` org.apache.commons.lang3.NotImplementedException` and method ` def getDate(dateTime: LocalDateTime)`